### DISCUSSION TITLE
How to move AutoModel to another computer and continue where it stopped?

### DISCUSSION THREAD

**Discussion by utahman** [keras-team/autokeras#1531] [Q&A]
Will it be enough to just pickle the AutoModel, then move the file to another computer, load it and call fit() to continue from where it stopped? Lets say if it did 15 trials on first system will continue from trial 16 on another system?
Or some files/folders must be transferred too?

**Comment by haifeng-jin** [comment]
No. We will need the files saved on disk in the project directory as well.

**Comment by stancsz** [comment]
similar to this ref, we'll need the generated directory to continue training on another machine:
https://github.com/keras-team/autokeras/discussions/1492#discussioncomment-587299
